src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PreReleaseService.cs

```diff
@@ -71,28 +66,52 @@ private DateTime GetEndTime(DateTime publishScheduled)
             return publishScheduled.AddMinutes(-_preReleaseOptions.MinutesBeforeReleaseTimeEnd);
         }
 
+        private DateTime GetPublishDayLenienceDeadline(DateTime publishScheduled)
```

Comment:
This'll probably work OK so long as each environment has the correct configuration provided.  On Prod and Pre-Prod, this'd be 570, on Dev it would probably be 120, and locally it would probably be 2.

What I was saying on our call thoug is that perhaps we can work this out dynamically based upon the first time the "930 job" runs on the given day that we're working out the lenience for.

An example of where we use this time is in `ReleaseService.ValidatePublishDate`.

Reply:
If we're doing the "subtract 60 minutes" logic on an environment like local or Dev, you'll probably end up with negative numbers too, FYI.

Reply:
The other super simple other approach could be to say that if you're already past midnight and the Release is due out today, simply allow PRA access up until the Release is published.  Literally "Is it past midnight (or the original PRA access window end) and has it been published yet?  No?  Then they can still see it!"

Reply:
Precisely - I'm hoping I've compensated for that with: 
`Math.Max(_preReleaseOptions.MinutesIntoPublishDayByWhichPublishingHasOccurred - 60, 0)`